src/applications/mhv/secure-messaging/components/AttachmentsList.jsx

```diff
@@ -135,7 +136,12 @@ const AttachmentsList = props => {
             status="success"
             onCloseEvent={handleSuccessAlertClose}
           >
-            <p className="vads-u-margin-bottom--0">File attached</p>
+            <h3
+              slot="headline"
+              className="vads-u-margin-bottom--0 vads-u-font-size--base vads-font-family-sans-serif"
+            >
+              File attached
+            </h3>
```

Comment:
Was this `h3` suggested by a11y? trying to see the rationale behind replacing `p` with heading tag